userguide/virtualmachines.rst

```diff
@@ -162,8 +162,8 @@ a Virtual Machine (VM) type are described in
    | 2        | Enable VNC         | checkbox       | Add a VNC remote connection. Requires *UEFI* booting.                                         |
    |          |                    |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
-   | 2        | Bind               | drop-down menu | VNC network interface. The primary interface is automatically selected, but secondary         |
-   |          |                    |                | interfaces can be chosen.                                                                     |
+   | 2        | Bind               | drop-down menu | VNC network interface. The primary interface is the default. A different interface IP address |
```

Comment:
It looks like we are just talking about which network interface is used, no IP address at all.  If that is correct, this text should not mention IP address.